engine/language-server/src/main/scala/org/enso/languageserver/libraries/LibraryApi.scala

```diff
@@ -0,0 +1,235 @@
+package org.enso.languageserver.libraries
+
+import io.circe.Json
+import io.circe.literal.JsonStringContext
+import org.enso.editions.{LibraryName, LibraryVersion}
+import org.enso.jsonrpc.{Error, HasParams, HasResult, Method, Unused}
+
+object LibraryApi {
```

Comment:
I assume the new API is documented.

Reply:
Yes, it is documented in `protocol-language-server.md#library-related-operations`.